pkg/util/util.go

```diff
@@ -0,0 +1,22 @@
+package util
+
+import (
+	"errors"
+	"strings"
+)
+
+// ExtractBusAndMgmtDevice extracts the busName and deviceName from a full device address (e.g. pci)
+// example 1: pci/65:0000.1   -> "pci", "65:0000.1",  nil
+// example 2: vdpa_sim        -> "",    "vdpa_sim",   nil
+// example 3: pci/65:0000.1/1 -> "",    "",           err
+func ExtractBusAndMgmtDevice(fullMgmtDeviceName string) (busName string, mgmtDeviceName string, err error) {
```

Comment:
This is specific to kernel vdpa framework so it should be inside `kvdpa`, not at the same level.

Reply:
:+1:

Reply:
Just curious why we need a module at all. Why not add this function to the kvdpa module?

Reply:
I thought utility functions can grow over time

Reply:
Forcing users to import two modules for a single function seems an overkill right now. Besides, even if utility functions grow so much that they don't fit a single file or "util-foo.go" files, this module cannot be used without kvdpa so I think maybe we could keep them in the same module? WDYT?

Reply:
Fair enough, is it ok to keep a separate file for util in the kvdpa module?

Reply:
Sure, we can have `util.go` inside kvpda module with this function (and any further utility func that can makes sense)